src/lib/loaders/api/extensionsLogger.ts

```diff
@@ -68,3 +68,10 @@ export const fetchLoggerRequestDone = requestID => () => {
 }
 
 export default extensionsLogger
+
+export const formatBytes = (bytes: number) => {
+  if (bytes < 1024) return bytes + " B"
+  else if (bytes < 1048576) return (bytes / 1024).toFixed(3) + " KB"
+  else if (bytes < 1073741824) return (bytes / 1048576).toFixed(3) + " MB"
+  else return (bytes / 1073741824).toFixed(3) + " GB"
```

Comment:
maybe this GB line should be "please dont"

Reply:
`GB!!!!!`

Reply:
"if you are reading this, we have a problem. Gravity is already down"